### DISCUSSION TITLE
brew install jenkins error

### DISCUSSION THREAD

**Discussion by ZhangQiJin** [Homebrew/discussions#198] [Everyday usage]
3teedeMacBook-Pro:~ 3tee$ brew install jenkins
Warning: You are using macOS 10.12.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.

==> Downloading https://mirrors.ustc.edu.cn/homebrew-bottles/bottles/autoconf-2.
Already downloaded: /Users/3tee/Library/Caches/Homebrew/downloads/f601e9d801a5ceb7e536b720e8e57d74e3bae6fb99f0e0c163c1f78b4b92e404--autoconf-2.69.sierra.bottle.4.tar.gz
==> Downloading https://raw.githubusercontent.com/Homebrew/formula-patches/2087f

curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection refused
Error: Failed to download resource "openjdk@11--patch"
Download failed: https://raw.githubusercontent.com/Homebrew/formula-patches/2087f9d035e568655d3f4240827e9cb7f31370da/openjdk%4011/xcode12.diff
3teedeMacBook-Pro:~ 3tee$

**Comment by SMillerDev** [comment]
> We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.

However in this case the problem is that your network connection rejected the connection to GitHub, I can't help you fix that since it isn't related to brew at all and network engineering isn't really my prime skill. You could try a different network or disabling firewalls to see if it helps.

**Reply by ZhangQiJin** [reply]
> > We (and Apple) do not provide support for this old version.
> > You will encounter build failures with some formulae.
> > Please create pull requests instead of asking for help on Homebrew's GitHub,
> > Twitter or any other official channels. You are responsible for resolving
> > any issues you experience while you are running this
> > old version.
> 
> However in this case the problem is that your network connection rejected the connection to GitHub, I can't help you fix that since it isn't related to brew at all and network engineering isn't really my prime skill. You could try a different network or disabling firewalls to see if it helps.

i try ，thanks
